sumo_mgmt.py

```diff
@@ -519,11 +619,42 @@ def print_collector_table(collectors, headings):
   table = AsciiTable(table_data)
   log('[INFO] %d total collectors\n%s' % (table_rows, table.table))
 
+def print_sources_table(collectors, headings):
+  '''
+  Given a set of headings and a list of Collectors, prints the attributes
```

Comment:
This description doesn't sound right.  It should talk about printing the source details, not the collector.